dist/debian/debian_files_gen.py

```diff
@@ -47,7 +48,13 @@ class DebianFilesTemplate(string.Template):
 shutil.copytree('dist/debian/debian', 'build/debian/debian')
 
 s = DebianFilesTemplate(changelog_template)
-changelog_applied = s.substitute(product=product, version=version, release=release, revision='1', codename='stable')
+now = datetime.datetime.now(tz=datetime.timezone.utc)
+changelog_applied = s.substitute(product=product,
```

Comment:
this was done on other packages ?

Reply:
i.e. adding timestamp ?

Reply:
(we assumed for long time, that the change log is mostly not touched)

Reply:
not yet. i created this change in hope to silence the warnings and to comply to the debian packaging policies. will take care of other packages when i have more free cycles.